### PR TITLE
feat: update person credential page

### PR DESCRIPTION
**Pull request by wadeking98**
UI improvements to the person credential page corresponding with these wireframes: https://xd.adobe.com/view/1281a7c0-bab1-4d5e-989e-a6760467020d-47d2/screen/06bb0c33-a927-47ce-a4d3-90cbbc0617c7/

# Flow 1 BC services card installed on other device:
personflow1

# Flow 2 BC services card installed on this device:
personcred-install
person-installed

### CODE REVIEW

File: app/src/screens/PersonCredential.tsx [hunk 1]
```diff
@@ -49,14 +41,44 @@ export default function PersonCredential() {
...
+      alignItems: 'center',
+      flexGrow: 1,
+      paddingHorizontal: 25,
+    },
+    itemSeparator: {
+      borderBottomWidth: 1,
+      borderBottomColor: ColorPallet.brand.primaryBackground,
+      marginHorizontal: 25,
+    },
+    test: {},
```
Comment (bryce-mcmath) [inline app/src/screens/PersonCredential.tsx]: Can probably remove this?

File: app/src/screens/PersonCredential.tsx [hunk 2]
```diff
@@ -65,82 +87,93 @@ export default function PersonCredential() {
...
-                    onPress={getBCServicesCardApp}
-                    hitSlop={hitSlop}
-                    testID={testIdWithKey('GetBCServicesCardApp')}
-                  >
-                    <Text style={{ ...TextTheme.normal, color: ColorPallet.brand.link }}>
-                      {t('PersonCredential.LinkDescription')}
-                    </Text>
-                  </TouchableOpacity>
+      <ScrollView style={styles.pageContent}>
+        <View style={{ ...styles.credentialCardContainer, display: 'flex', alignItems: 'center' }}>
```
Comment (bryce-mcmath) [inline app/src/screens/PersonCredential.tsx]: Can we put these styles in the stylesheet?

File: app/src/screens/PersonCredential.tsx [hunk 3]
```diff
@@ -65,82 +87,93 @@ export default function PersonCredential() {
...
+                onPress={getBCServicesCardApp}
+                accessibilityLabel={t('PersonCredential.InstallApp')}
+                testID={testIdWithKey('InstallApp')}
+                title={t('PersonCredential.InstallApp')}
+              ></Button>
+              <TouchableOpacity
+                onPress={() => setAppInstalled(true)}
+                accessibilityLabel={t('PersonCredential.AppOnOtherDevice')}
+                testID={testIdWithKey('AppOnOtherDevice')}
+                style={{ display: 'flex', alignItems: 'center', marginTop: 10 }}
```
Comment (bryce-mcmath) [inline app/src/screens/PersonCredential.tsx]: Can we put these in the stylesheet as well

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
Looks great, some minor changes requested

File: app/src/screens/PersonCredential.tsx (hunk 1 above)
Comment (wadeking98) [reply app/src/screens/PersonCredential.tsx]: good catch, done

**Comment by knguyenBC** [comment]
The blocks of content should bleed edge to edge. Also, the buttons change height when you tap on them. Can they be fixed?

**Comment by wadeking98** [comment]
> The blocks of content should bleed edge to edge. Also, the buttons change height when you tap on them. Can they be fixed?

got it, this is how it looks after the fix:
fix

**Review by bryce-mcmath — APPROVED** [review]
Nicely done

**Comment by knguyenBC** [comment]
SO QUICK. you guys rock. looks good.

**Pull request merged**